core/bus/camel/src/main/java/org/switchyard/bus/camel/CamelExchange.java

```diff
@@ -171,9 +174,29 @@ public void send(Message message) {
             getContext().setProperty(Exchange.MESSAGE_ID, camelMsg.getMessageId());
             initOutContentType();
         }
-
+        copyProperties(camelMsg);
```

Comment:
This was required Tomo. Without this piece of code, the  properties were not propagated.